rules/linux/file_event/file_event_lnx_persistence_cron_files.yml

```diff
@@ -1,12 +1,13 @@
 title: Persistence Via Cron Files
 id: 6c4e2f43-d94d-4ead-b64d-97e53fa2bd05
 status: test
-description: Detects creation of cron file or files in Cron directories which could indicates potential persistence.
+description: |
+    Detects the creation of cron file or files in Cron directories which could indicates potential persistence.
```

Comment:
We should add more verbose description, describing that not any creation is malicious. Instead it should be investigated...etc.